goopicsi.go

```diff
@@ -126,6 +126,50 @@ func NVMeControllerConnect(request *pb.NVMfRemoteController) (*pb.NVMfRemoteCont
 	return &pb.NVMfRemoteControllerConnectResponse{}, nil
 }
 
+// NVMeControllerList lists all the connections to the remote NVMf controller
+func NVMeControllerList() (*pb.NVMfRemoteControllerListResponse, error) {
```

Comment:
Please change this to some other `struct` instead of returning the proto messages

Reply:
Fixed